### PR TITLE
changed multiple post requests to one

### PR DESCRIPTION
**Pull request by JupitersLB**

### CODE REVIEW

File: app/controllers/api/v1/user_availabilities_controller.rb [hunk 1]
```diff
@@ -16,10 +16,19 @@ def destroy
     render json: timeblock
   end
 
+  def multiple
+    ids = params[:time_block_ids]
```
Comment (Naokimi) [inline app/controllers/api/v1/user_availabilities_controller.rb]: it's better to call the strong params

File: app/controllers/api/v1/user_availabilities_controller.rb [hunk 2]
```diff
@@ -16,10 +16,19 @@ def destroy
...
+    p ids
+    ids.each do |id|
+      new_avail = UserAvailability.new(user: current_user, time_block_id: id)
+      new_avail.save
```
Comment (Naokimi) [inline app/controllers/api/v1/user_availabilities_controller.rb]: why not directly `UserAvailability.create()` ? 
you could then fit in one line

Comment (JupitersLB) [reply app/controllers/api/v1/user_availabilities_controller.rb]: bad habit I guess

File: app/controllers/api/v1/user_availabilities_controller.rb (hunk 1 above)
Comment (JupitersLB) [reply app/controllers/api/v1/user_availabilities_controller.rb]: Fixed in the latest commit

File: app/controllers/api/v1/user_availabilities_controller.rb [hunk 3]
```diff
@@ -16,10 +16,21 @@ def destroy
     render json: timeblock
   end
 
+  def multiple
+    ids = time_block_ids_params
+    ids.each do |id|
+      UserAvailability.create(user: current_user, time_block_id: id)
+    end
```
Comment (Naokimi) [inline app/controllers/api/v1/user_availabilities_controller.rb]: should have been more specific 😅 
I think you can do `availability_params[:time_block_ids]` and that will already be a strong param

also if it's a single line better to write as `ids.each { |id| UserAvailability.create(user: current_user, time_block_id: id) }`

**Review by Naokimi — COMMENTED** [review]
I'd recommend implementing Rubocop

File: app/controllers/api/v1/user_availabilities_controller.rb (hunk 3 above)
Comment (JupitersLB) [reply app/controllers/api/v1/user_availabilities_controller.rb]: > availability_params[:time_block_ids]

This didn't work.

**Pull request merged**